lib/Twig/TokenParser/Block.php

```diff
@@ -28,7 +28,7 @@ public function parse(Twig_Token $token)
         $stream = $this->parser->getStream();
         $name = $stream->expect(Twig_Token::NAME_TYPE)->getValue();
         if ($this->parser->hasBlock($name)) {
-            throw new Twig_Error_Syntax(sprintf("The block '$name' has already been defined line %d", $this->parser->getBlock($name)->getLine()), $stream->getCurrent()->getLine(), $stream->getFilename());
+            throw new Twig_Error_Syntax(sprintf("The block '%s' has already been defined line %d", $name, $this->parser->getBlock($name)->getLine()), $stream->getCurrent()->getLine(), $stream->getFilename());
```

Comment:
I've added the dots to the exception messages.
However, I know you don't like a lot of small changes for little gain. So please let me know if you want to go down the road of adding the dots, otherwise I remove the last the commit from the PR.

(https://github.com/twigphp/Twig/commit/aa7c53ae0a4c1e02931cb6eac034bc9e13191b5a)